<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test_scale_container</class>
 <widget class="QWidget" name="test_scale_container">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test_scale_container</string>
  </property>
  <widget class="QWidget" name="scale_animation_container" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="v_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_0">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(199, 255, 108);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(90, 73, 255);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
